Description: Old nouveau driver is now called libdrm_nouveau1 at the latest libdrm package 
 Upstream nouveau 2 driver is now just called libdrm_nouveau, and at Ubuntu the libdrm
 package is still maintaining the older library, but as libdrm_nouveau1
Forwarded: not-needed
Author: Ricardo Salveti <rsalveti@linaro.org>

Index: plymouth-0.8.4/configure.ac
===================================================================
--- plymouth-0.8.4.orig/configure.ac	2012-08-16 02:47:21.845313178 -0300
+++ plymouth-0.8.4/configure.ac	2012-08-16 02:49:41.150003950 -0300
@@ -148,7 +148,7 @@
 AM_CONDITIONAL(ENABLE_LIBDRM_NOUVEAU,  [test "$enable_libdrm_nouveau" = yes])
 
 if test x$enable_libdrm_nouveau = xyes; then
-  PKG_CHECK_MODULES(DRM_NOUVEAU, [libdrm libdrm_nouveau])
+  PKG_CHECK_MODULES(DRM_NOUVEAU, [libdrm libdrm_nouveau1])
   OLD_CFLAGS="$CFLAGS"
   CFLAGS="$CFLAGS $DRM_NOUVEAU_CFLAGS"
   AC_MSG_CHECKING([if nouveau_drm.h is in include path])
